torch/CMakeLists.txt

```diff
@@ -431,6 +431,9 @@ endif()
 # Preserve XPU arch flags
 if(USE_XPU)
   string(REPLACE "," " " _ARCH_FLAGS_readable "${TORCH_XPU_ARCH_LIST}")
+  if("${TORCH_XPU_ARCH_LIST}" STREQUAL "")
+    set(_ARCH_FLAGS_readable "\\\"\\\"")
```

Comment:
Change the empty string to `""` to avoid string conversion failure in `C10_STRINGIZE` function.

Reply:
I think a better way is to promote `C10_STRINGIZE` to support the empty string `""`, like
```cpp
#define C10_STRINGIZE_IMPL(...) #__VA_ARGS__
#define C10_STRINGIZE(...) C10_STRINGIZE_IMPL(__VA_ARGS__)
```

Reply:
I tried. It doesn't work. Still have the parsing issue.
![image](https://github.com/user-attachments/assets/81f64bc2-c2a1-4fc6-aba8-ecadb0038857)